From 448828a38b9b5a178a05d367ea61f2bf795c4c1b Mon Sep 17 00:00:00 2001
From: jbelloua <jean-marie.bellouard@intel.com>
Date: Tue, 18 Sep 2012 16:10:29 +0200
Subject: [PATCH 13/49] [PORT FROM MAIN] Revert "[AUDIO] Select Widi output
 device first for media streams."

BZ: 57447

need to remove this commit and add patch 66939
wired headset should be used before widi if present: see CAP CPA_TCD-80223

This reverts commit 9680edb33bc86cb7209c38214ce8df0f9f9c5701.

Change-Id: I3300f65025443877277cb363d50f0604a3cff069
Origin-Change-Id: Ibc8b77e62e6e86aae6cb3365a6ab828eb27db36d
Signed-off-by: jbelloua <jean-marie.bellouard@intel.com>
Reviewed-on: http://android.intel.com:8080/77906
Reviewed-by: Barthes, FabienX <fabienx.barthes@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 audio/AudioPolicyManagerBase.cpp |    7 +------
 1 files changed, 1 insertions(+), 6 deletions(-)

diff --git a/audio/AudioPolicyManagerBase.cpp b/audio/AudioPolicyManagerBase.cpp
index 59c2d35..b81e0ad 100644
--- a/audio/AudioPolicyManagerBase.cpp
+++ b/audio/AudioPolicyManagerBase.cpp
@@ -2246,13 +2246,8 @@ audio_devices_t AudioPolicyManagerBase::getDeviceForStrategy(routing_strategy st
     case STRATEGY_MEDIA: {
 
         uint32_t device2 = 0;
-
-	if(mForceUse[AudioSystem::FOR_MEDIA] == AudioSystem::FORCE_SPEAKER) {
-            LOGV("getDeviceForStrategy() Out. strategy: MEDIA, device: SPEAKER for forceuse.");
-            return AUDIO_DEVICE_OUT_SPEAKER;
-        }
         if (mHasA2dp && (mForceUse[AudioSystem::FOR_MEDIA] != AudioSystem::FORCE_NO_BT_A2DP) &&
-            (getA2dpOutput() != 0) && !mA2dpSuspended) {
+                (getA2dpOutput() != 0) && !mA2dpSuspended) {
             device2 = mAvailableOutputDevices & AudioSystem::DEVICE_OUT_BLUETOOTH_A2DP;
             if (device2 == 0) {
                 device2 = mAvailableOutputDevices & AudioSystem::DEVICE_OUT_BLUETOOTH_A2DP_HEADPHONES;
-- 
1.7.9

